internal/infra/auth0/jwt_validator.go

```diff
@@ -0,0 +1,88 @@
+package auth0
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/auth0/go-jwt-middleware/v2/jwks"
+	"github.com/auth0/go-jwt-middleware/v2/validator"
+	"net/url"
+	"time"
+)
+
+type JwtValidator interface {
+	ValidateToken(context.Context, string) (*JwtToken, error)
+}
+
+// jwt-validatorのラップ構造体
+// tokenの型も独自定義型に詰め替える
+type jwtValidator struct {
+	auth0domain, auth0audience string
+}
+
+func NewJwtValidator(auth0domain string, auth0audience string) JwtValidator {
+	return &jwtValidator{auth0domain: auth0domain, auth0audience: auth0audience}
+}
```

Comment:
nits
[このルール](https://zenn.dev/keitakn/articles/go-naming-rules#%E9%A0%AD%E6%96%87%E5%AD%97%E3%82%84%E9%A0%AD%E6%96%87%E5%AD%97%E3%82%92%E3%81%A8%E3%81%A3%E3%81%9F%E5%90%8D%E5%89%8D%E3%81%AE%E5%8D%98%E8%AA%9E%E3%81%AE%E5%91%BD%E5%90%8D%E3%81%AB%E3%81%A4%E3%81%84%E3%81%A6)に則るとこんな感じが良いですね

```suggestion
type JWTValidator interface {
	ValidateToken(context.Context, string) (*JWTToken, error)
}

// jwt-validatorのラップ構造体
// tokenの型も独自定義型に詰め替える
type jwtValidator struct {
	auth0domain, auth0audience string
}

func NewJWTValidator(auth0domain string, auth0audience string) JWTValidator {
	return &jwtValidator{auth0domain: auth0domain, auth0audience: auth0audience}
}
```

Reply:
> 元々略語として浸透している単語は一貫した大文字と小文字

大文字統一ってこういう意図だったんですね。「privateとpublicを分ける為〜」以上のガイドラインを知りませんでした、、、

Reply:
修正しました！
[8b8cfeb](https://github.com/dainoguchi/bstodo-api/pull/6/commits/8b8cfeba47509804d003bacd308e109f8d57f67e)